src/Documenter.jl

```diff
@@ -316,6 +317,18 @@ documentation. By default this value is set to `"master"`.
 **`devurl`** the folder that in-development version of the docs will be deployed.
 Defaults to `"dev"`.
 
+**`selector`** determines content and order of the resulting version selector in
+the generated html. The following entries are valied in the `selector` vector:
```

Comment:
Just bikeshedding here now -- since the `selector` also defined which symlinks (i.e. which public URLs) get deployed, maybe it should be called something like `versions`? It's not only the selector.

I would also mention here that it creates the URLs the person reading the docs sees and should use.

Reply:
I like `versions`.